app/views/emergency/newCookieIssued.scala.html

```diff
@@ -0,0 +1,23 @@
+@()
```

Comment:
The file name is `newCookieIssued` but it looks to be request form so perhaps it could be called `requestNewCookie`?